front/app/api/app_configuration/types.ts

```diff
@@ -47,8 +47,8 @@ export type IAppConfigurationSettingsCore = {
   topics_term?: Multiloc;
   topic_term?: Multiloc;
   authentication_token_lifetime_in_days: number;
-  maximum_admins_number: number;
-  maximum_project_moderators_number: number;
+  maximum_admins_number: number | null;
```

Comment:
Probably, now it can be `undefined` too. You can test it with `con = AppConfiguration.last; con.settings['core'].delete('maximum_project_moderators_number'); con.save!` in the rails console (`make rails-console`)

Reply:
What would the process look like for it to be undefined in the user journey?

Reply:
I think the same as `null` would be fine.

Reply:
In that case, we can ignore undefined because technically, undefined would only be possible through the console and that wouldn't be an ideal flow

Reply:
For now, it's also `undefined` if a new tenant was just created (though it may be changed in the future). So, it can happen in production too.